### PR TITLE
Update materials for April 23

### PR DESCRIPTION
**Pull request by eugene123tw**
The materials on April 23 include gradient descent, Taylor series, Fixed-Point Iteration and Learning rate adjustment.

### CODE REVIEW

**Comment by eugene123tw** [comment]
Update notes on April 30
